core/src/test/java/org/apache/calcite/materialize/LatticeSuggesterTest.java

```diff
@@ -721,6 +721,36 @@ private void checkDerivedColumn(Lattice lattice, List<String> tables,
     checkFoodmartSimpleJoin(CalciteAssert.SchemaSpec.FAKE_FOODMART);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6374">[CALCITE-6374]
+   * LatticeSuggester throw NullPointerException when agg call covered with cast </a>. */
+  @Test void testCastExpression() throws Exception {
```

Comment:
Maybe the name testCastExpression is not clear enough. Is there a clearer way to use it?

Reply:
Adjusted test method name.